src/main/java/org/candlepin/subscriptions/capacity/PoolIngressController.java

```diff
@@ -29,15 +29,15 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.function.Function;
 import java.util.stream.Collectors;
 import javax.transaction.Transactional;
+
```

Comment:
Make sure to run `./gradlew spotlessApply` before checking in (or run it while doing a clean build), this blank line and the indenting on line 64 would be corrected.

Reply:
I did run that. That validation is part of the check that runs for every commit. Not sure why it did not remove this line.